jetty9/pom.xml

```diff
@@ -27,12 +27,6 @@
       <version>${project.version}</version>
       <type>tar.gz</type>
     </dependency>
-    <dependency>
```

Comment:
I just tried to hit the site without a Host header and it failed because it is a virtual host sharing an IP address.  So no host header will always result in a failed request.